test.js

```diff
@@ -194,6 +194,16 @@ test('handles infinite stream', async t => {
 	await t.throwsAsync(setup(infiniteIteration, {maxBuffer: 1}), {instanceOf: MaxBufferError});
 });
 
+test.serial('handles streams larger than arrayBuffer max length', async t => {
+	t.timeout(BIG_TEST_DURATION);
+	const chunkSize = 2 ** 16;
+	const chunkCount = Math.floor(BufferConstants.MAX_LENGTH / chunkSize * 2);
```

Comment:
`ArrayBuffers` have the same max length in Node.js as `Buffer`, i.e. `2 ** 32 - 1`.